packages/dds/tree/src/test/simple-tree/treeContext.spec.ts

```diff
@@ -29,12 +29,11 @@ describe("TreeBranch", () => {
 		return view;
 	}
 
-	{
-		// Test that branching from a TreeView returns a typed view (as opposed to an untyped context).
+	it("Test that branching from a TreeView returns a typed view (as opposed to an untyped context)", () => {
```

Comment:
This was running the schema validation during test enumeration which made debugging tests annoying, so I made it a test.